src/aui-form-builder/tests/unit/js/aui-form-builder-page-manager-tests.js

```diff
@@ -260,12 +260,72 @@ YUI.add('aui-form-builder-pages-tests', function(Y) {
                     Y.Assert.areEqual('An aditional info about this page', descriptionNode.get('placeholder'));
                 });
             });
+        },
+
+        'should initialize with pagination tabs if the default option was replaced': function() {
+            this.createFormBuilderPageManager({
+                activePageNumber: 1,
+                pageHeader: '#header',
+                pagesQuantity: 1,
+                paginationContainer: '#pages',
+                tabviewContainer: '#tabs',
+                mode: 'tabs'
+            });
+
+            Y.Assert.isFalse(Y.one('.tabbable-content').hasClass('hide'));
+            Y.Assert.isTrue(Y.one('.pagination-content').hasClass('hide'));
+        },
+
+        'should not mode attribute accept values not equal to \'pagination\' or \'tabs\'': function() {
+            var pages;
+
+            pages = this.createFormBuilderPageManager({
+                activePageNumber: 1,
+                pageHeader: '#header',
+                pagesQuantity: 1,
+                paginationContainer: '#pages',
+                tabviewContainer: '#tabs',
+                mode: 'numbers'
+            });
+
+            Y.Assert.areNotEqual(pages.get('mode'), 'numbers');
+        },
+
+        'should keep the indication of the page selected after page mode change from pagination to tabs': function() {
```

Comment:
Based on this description it's not clear what this test is doing, could you update this description adding more details?